hasura/metadata/databases/default/tables/public_membership.yaml

```diff
@@ -56,3 +60,11 @@ delete_permissions:
   - role: service
     permission:
       filter: {}
+  - role: user
```

Comment:
can be in a separate PR, but we should let ADMINs change all the way to ADMIN (including downgrading)

Reply:
same for deletion below